core/templates/services/user.service.ts

```diff
@@ -108,6 +108,15 @@ export class UserService {
       return data.default_dashboard;
     });
   }
+
+  async getCanUserAccessTopicsAndSkillsDashboard(): Promise<boolean> {
```

Comment:
This doesn't read properly .. shouldn't it be canUserAccess...?

Reply:
Done.